### PR TITLE
Minor toolbar fixes

### PR DESCRIPTION
**Pull request by mthouve**
This fixes the toolbar title CSS class and adds a sample usage of it to the demo site.

### CODE REVIEW

File: addon/components/mdc-toolbar/title.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-import { MDCToolbarFoundation } from '@material/toolbar';
-
-const { strings } = MDCToolbarFoundation;
 
 export default Ember.Component.extend({
   //region Ember Hooks
   layout,
-  classNames: [`${strings.TITLE_SELECTOR}`],
+  classNames: ['mdc-toolbar__title'],
+  classNameBindings: ['fixed:mdc-toolbar__title--fixed'],
```
Comment (Kerrick) [inline addon/components/mdc-toolbar/title.js]: In [the docs](https://material.io/components/web/catalog/toolbar/) I didn't see anything about the class `mdc-toolbar__title--fixed` -- only about `mdc-toolbar__title` and `mdc-toolbar--fixed`.

File: addon/components/mdc-toolbar/title.js [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 import layout from '../../templates/components/mdc-toolbar/title';
-import { MDCToolbarFoundation } from '@material/toolbar';
-
-const { strings } = MDCToolbarFoundation;
 
 export default Ember.Component.extend({
   //region Ember Hooks
   layout,
-  classNames: [`${strings.TITLE_SELECTOR}`],
+  classNames: ['mdc-toolbar__title'],
```
Comment (Kerrick) [inline addon/components/mdc-toolbar/title.js]: :+1: Good add, but please import this class from the Foundation ([here](https://github.com/material-components/material-components-web/blob/21db79416a707cdc13d06c616f276ea5e4e73a3e/packages/mdc-toolbar/constants.js#L28)), as we do in `{{mdc-toolbar}}`.

https://github.com/secondstreet/ember-material-components-web/blob/bf27d26cac0cf141fae9c4561b8980a1186ef95e/addon/components/mdc-toolbar.js#L10

File: addon/components/mdc-toolbar/title.js (hunk 1 above)
Comment (mthouve) [reply addon/components/mdc-toolbar/title.js]: Good catch!

File: addon/components/mdc-toolbar/title.js (hunk 2 above)
Comment (mthouve) [reply addon/components/mdc-toolbar/title.js]: @Kerrick are you sure? Currently, this line change fixes the bug. The `strings.TITLE_SELECTOR` value comes with a leading period.  In other components such as [here](https://github.com/secondstreet/ember-material-components-web/blob/master/addon/components/mdc-toolbar/section.js#L19) or [here](https://github.com/secondstreet/ember-material-components-web/blob/master/addon/components/mdc-toolbar/row.js#L9) we hard-coded the class name.

Comment (Kerrick) [reply addon/components/mdc-toolbar/title.js]: Oh, good catch. It was in strings as a selector, not in cssClasses as a classname.

**Comment by Kerrick** [comment]
@mthouve If you can rebase onto latest `master` this PR looks good to go! :+1:

**Comment by Kerrick** [comment]
Thank you so much for the PR! This will ship out with v0.0.37 👍

**Pull request merged**